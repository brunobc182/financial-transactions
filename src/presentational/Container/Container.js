import React from 'react';
import { node } from 'prop-types';
import styled from 'styled-components';

const Container = ({ children }) => <Wrapper>{children}</Wrapper>;

const Wrapper = styled.div`
  position: absolute;
  top: 20%;
  left: 50%;
  transform: translate(-50%, -50%);
`;

Container.propTypes = {
  children: node.isRequired,
};

export default Container;
