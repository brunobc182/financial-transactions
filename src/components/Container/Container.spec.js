import React from 'react';
import renderer from 'react-test-renderer';
import Container from './Container';

describe('Container', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<Container>Children</Container>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
