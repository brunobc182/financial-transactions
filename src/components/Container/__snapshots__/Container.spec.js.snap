// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container should render correctly 1`] = `
<div
  className="sc-bdVaJa crlonA"
>
  Children
</div>
`;
